composer.json

```diff
@@ -12,6 +12,7 @@
         }
     ],
     "autoload": {
+        "files": ["src/functions_include.php"],
```

Comment:
The classloader doesn't autoload functions, so I think it's necessary.
The only recent change for functions I can think of is the addition of `use function` in 5.6, but that still won't autoload them.